### PR TITLE
Simplify private testing for templates

### PR DESCRIPTION
**Pull request by scottcwilson**
Current code does not work.  This works as tested with 

http://localhost:8888/gh_demo_158/index.php?main_page=index&t=classic

then 

http://localhost:8888/gh_demo_158/index.php?main_page=index&t=off

Suggested for backport with low priority.

### CODE REVIEW

**Comment by scottcwilson** [comment]
Instructions on using this feature: 
https://docs.zen-cart.com/user/template/private_testing/

File: includes/init_includes/init_templates.php [hunk 1]
```diff
@@ -33,19 +33,14 @@
  * Allow admins to switch templates using &t= URL parameter
  */
 if (zen_is_whitelisted_admin_ip()) {
-    $templates = array();
-    foreach($result as $row) {
-        $templates[] = $row['template_dir'];
-    }
```
Comment (drbyte) [inline includes/init_includes/init_templates.php:39]: Excluding the check of this array means the only check for templates is against the filesystem.

I intended it to require that the template be registered in the admin in order for catalog-side to be able to switch to it.

Comment (scottcwilson) [reply includes/init_includes/init_templates.php:39]: This seems like a really confusing and hard to figure out additional step.  We already have the protection that only admins can do this - why not make it easy for them?

Comment (drbyte) [reply includes/init_includes/init_templates.php:39]: I'll concede.

Please also remove `and that it matches an available choice` from the comment.

And change the commit and PR title from "Fix..." to "Simplify..." or "Change...".

Comment (scottcwilson) [reply includes/init_includes/init_templates.php:39]: Ugh sorry for the double commit - please squash.

Comment (drbyte) [reply includes/init_includes/init_templates.php:39]: I'd hoped you would have updated the commit title as well (part of the amend+forcepush), not just the PR title.

**Comment by scottcwilson** [comment]
Please consider for backport - this is a great feature I was unaware of prior to working on the doc page reference above.

**Pull request merged**